src/Sylius/Bundle/ApiBundle/CommandHandler/AddProductReviewHandler.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\CommandHandler;
+
+use Sylius\Bundle\AdminApiBundle\Model\UserInterface;
+use Sylius\Bundle\ApiBundle\Command\AddProductReview;
+use Sylius\Bundle\ApiBundle\Context\UserContextInterface;
+use Sylius\Bundle\ApiBundle\Provider\CustomerProviderInterface;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\ShopUserInterface;
+use Sylius\Component\Core\Repository\ProductRepositoryInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\Component\Review\Model\ReviewInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+/** @experimental */
+final class AddProductReviewHandler implements MessageHandlerInterface
+{
+    /** @var CustomerProviderInterface */
+    private $customerProvider;
+
+    /** @var RepositoryInterface */
+    private $productReviewRepository;
+
+    /** @var FactoryInterface */
+    private $productReviewFactory;
+
+    /** @var ProductRepositoryInterface */
+    private $productRepository;
+
+    public function __construct(
+        CustomerProviderInterface $customerProvider,
+        RepositoryInterface $productReviewRepository,
+        FactoryInterface $productReviewFactory,
+        ProductRepositoryInterface $productRepository
```

Comment:
I would be for this order:
```suggestion
        FactoryInterface $productReviewFactory,
        RepositoryInterface $productReviewRepository,
        ProductRepositoryInterface $productRepository,
        CustomerProviderInterface $customerProvider
```
here and in variables definitions and assignments